packages/curriculum-compiler-json/index.js

```diff
@@ -65,6 +66,7 @@ function compileInsight(ast) {
 function getCompiler(type) {
   let compileSync;
   switch (type) {
+    case contentTypes.EXERCISE:
```

Comment:
so you have the exercise compiler use question methods but here it equates to an insight

Reply:
Well, the spec calls for the same compiler to be applicable to [exercise](https://github.com/enkidevs/curriculum/blob/master/javascript/core/functions-ii/js-practice-this.md) and [insight](https://github.com/enkidevs/curriculum/blob/master/javascript/core/functions-ii/check-if-a-function-exists.md) file types... On line 24 I invoke the new compiler if it's an `## Exercise` section